client/src/components/App.js

```diff
@@ -14,28 +15,35 @@ class App extends Component {
     };
   }
   componentDidMount() {
-    fetch("/api/users")
-      .then(res => res.json())
-      .then(users => this.setState({ users }))
-      .catch(err => console.log(err));
-    fetch("/api/matches")
-      .then(res => res.json())
-      .then(matches => this.setState({ matches }))
-      .catch(err => console.log(err));
-    fetch("/api/teams")
-      .then(res => res.json())
-      .then(teams => this.setState({ teams }))
-      .catch(err => console.log(err));
+    Promise.all([
+      fetch("/api/users")
+        .then(res => res.json())
```

Comment:
you don't need the `.then` here if you are using `Promise.all` you would only need to do `.then` on the `Promise.all` itself `Promise.all(fetch("/api/users"), fetch("/api/matches"), fetch("/api/teams")).then(apiResultsObject => do something`

Reply:
Done.